sim/common/sod/item_effects/phase_5.go

```diff
@@ -20,6 +31,65 @@ func init() {
 
+	itemhelpers.CreateWeaponEquipProcDamage(DrakeTalonCleaver, "Drake Talon Cleaver", 1.0, 467167, core.SpellSchoolPhysical, 300, 0, 0.0, core.DefenseTypeMelee) // TBD confirm 1 ppm in SoD
+
+	itemhelpers.CreateWeaponEquipProcDamage(HaldberdOfSmiting, "Halberd of Smiting", 0.5, 467819, core.SpellSchoolPhysical, 452, 224, 0.0, core.DefenseTypeMelee)         // TBD does this work as phantom strike?, confirm 0.5 ppm in SoD
+	itemhelpers.CreateWeaponEquipProcDamage(HaldberdOfSmitingBloodied, "Halberd of Smiting", 0.5, 467819, core.SpellSchoolPhysical, 452, 224, 0.0, core.DefenseTypeMelee) // TBD does this work as phantom strike?, confirm 0.5 ppm in SoD
+
+	itemhelpers.CreateWeaponCoHProcDamage(JekliksCrusher, "Jeklik's Crusher", 4.0, 467642, core.SpellSchoolPhysical, 200, 20, 0.0, core.DefenseTypeMelee)
+	itemhelpers.CreateWeaponCoHProcDamage(JekliksCrusherBloodied, "Jeklik's Crusher", 4.0, 467642, core.SpellSchoolPhysical, 200, 20, 0.0, core.DefenseTypeMelee)
+
+	core.NewItemEffect(TheUntamedBlade, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		strengthAura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Untamed Fury",
+			ActionID: core.ActionID{SpellID: 23719},
+			Duration: time.Second * 8,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				character.AddStatDynamic(sim, stats.Strength, 300)
+			},
+			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+				character.AddStatDynamic(sim, stats.Strength, -300)
+			},
+		})
+
+		core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+			Name:              "The Untamed Blade (Strength)",
+			Callback:          core.CallbackOnSpellHitDealt,
+			Outcome:           core.OutcomeLanded,
+			ProcMask:          core.ProcMaskMelee,
+			SpellFlagsExclude: core.SpellFlagSuppressWeaponProcs,
+			PPM:               1, // Estimated based on data from WoW Armaments Discord
+			Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+				strengthAura.Activate(sim)
+			},
+		})
+	})
+
+	core.NewItemEffect(HammerOfTheLightbringer, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		blazefuryTriggerAura(character, 465412, core.SpellSchoolHoly, 4)
+		crusadersZealAura465414(character)
+	})
+
+	// https://www.wowhead.com/classic/item=229749/truthbearer
+	// Chance on hit: Increases damage done by 15 and attack speed by 30% for 8 sec.
+	// TODO: Proc rate assumed and needs testing
+	core.NewItemEffect(Truthbearer1H, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		crusadersZealAura465414(character)
+	})
+
+	// https://www.wowhead.com/classic/item=229806/truthbearer
+	// Chance on hit: Increases damage done by 15 and attack speed by 30% for 8 sec.
+	// TODO: Proc rate assumed and needs testing
+	core.NewItemEffect(Truthbearer2H, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		blazefuryTriggerAura(character, 465412, core.SpellSchoolHoly, 4)
+		crusadersZealAura465414(character)
+	})
```

Comment:
Since these 2 are paladin-only can you put them in `sim/paladin/items.go`?

Reply:
Totally missed that in the shockadin PR

Reply:
moved